crates/red_knot_python_semantic/Cargo.toml

```diff
@@ -35,6 +35,7 @@ smallvec = { workspace = true }
 static_assertions = { workspace = true }
 test-case = { workspace = true }
 memchr = { workspace = true }
+dir-test = "0.3.0"
```

Comment:
Our preferred style for pinning dependencies is to put the pins in the `Cargo.toml` file in the repository root. So you'd put `dir-test = "0.3.0"` in the `Cargo.toml` file in the repository root, and in this `Cargo.toml` file you'd just put `dir-test = { workspace = true }`